### PR TITLE
FF127/125 Clipboard API release notes/expr feature

### PR DESCRIPTION
**Pull request by hamishwillee**
FF127 enables the last parts of the Clipboard API including 
- `ClipboardItem` interface
- `read()`/`write()` methods  of `Clipboard` https://bugzilla.mozilla.org/show_bug.cgi?id=1887845. 
- `ClipboardItem.supports` in https://bugzilla.mozilla.org/show_bug.cgi?id=1858788 (though I haven't mentioned this because it is part of `ClipboardItem`, and all of that is coming available at same time.

FF125 added support for `Clipboard.readText()` in https://bugzilla.mozilla.org/show_bug.cgi?id=1877400

This updates the release notes for FF125 and 127 and removes the notes from experimental features.
This is part of #33565

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/125/index.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ No notable changes.
   These set the [`clonable`](/en-US/docs/Web/API/ShadowRoot/clonable) property of a [`ShadowRoot`](/en-US/docs/Web/API/ShadowRoot) that has been created declaratively using the [`<template>`](/en-US/docs/Web/HTML/Element/template) element.
   ([Firefox bug 1880188](https://bugzil.la/1880188)).
 
+- The [`readText()`](/en-US/docs/Web/API/Clipboard/readText) method of the [`Clipboard`](/en-US/docs/Web/API/Clipboard/readText) interface is now supported for asynchronously reading text from the system clipboard.
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/125/index.md:68]:
```suggestion
- The [`readText()`](/en-US/docs/Web/API/Clipboard/readText) method of the [`Clipboard`](/en-US/docs/Web/API/Clipboard) interface is now supported for asynchronously reading text from the system clipboard.
```

File: files/en-us/mozilla/firefox/releases/125/index.md [hunk 2]
```diff
@@ -65,6 +65,9 @@ No notable changes.
...
+  A paste context menu will appear for the user to confirm when reading clipboard data not provided by the same-origin page. ([Firefox bug 1877400](https://bugzil.la/1877400)).
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/125/index.md:69]:
```suggestion
  A paste context menu will appear for the user to confirm when reading clipboard data is not provided by the same-origin page. ([Firefox bug 1877400](https://bugzil.la/1877400)).
```
or 

```suggestion
  When reading clipboard data that is not provided by the same-origin page, a paste context menu will appear for the user to confirm. ([Firefox bug 1877400](https://bugzil.la/1877400)).
```

File: files/en-us/mozilla/firefox/releases/127/index.md [hunk 3]
```diff
@@ -38,6 +38,8 @@ This article provides information about the changes in Firefox 127 that affect d
 
 ### APIs
 
+- Firefox now fully supports the async {{domxref('Clipboard API')}}. This release enables the {{domxref('ClipboardItem')}} interface, and the [`read()`](/en-US/docs/Web/API/Clipboard/read) and [`write()`](/en-US/docs/Web/API/Clipboard/write) methods of the {{domxref('Clipboard')}} interface. ([Firefox bug 1887845](https://bugzil.la/1887845),[Firefox bug 1858788](https://bugzil.la/1858788)).
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/127/index.md:41]: This is not incorrect but we usually don't mention "Firefox". An alternative phrasing could be:
```suggestion
- The async {{domxref('Clipboard API')}} is now fully supported. The {{domxref('ClipboardItem')}} interface, along with the [`read()`](/en-US/docs/Web/API/Clipboard/read) and [`write()`](/en-US/docs/Web/API/Clipboard/write) methods of the {{domxref('Clipboard')}} interface, have been enabled. ([Firefox bug 1887845](https://bugzil.la/1887845),[Firefox bug 1858788](https://bugzil.la/1858788)).
```

**Review by dipikabh — APPROVED** [review]
Just a few nits again :) and a stylistic suggestion. Approving.
Thanks, @hamishwillee!

File: files/en-us/mozilla/firefox/releases/125/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/125/index.md:68]: Thanks. Good eyes!

**Comment by hamishwillee** [comment]
Thanks very much @dipikabh

**Pull request merged**